### ISSUE TITLE
No comments

### ISSUE THREAD

**Issue by Gwizdo51** [javascript-tutorial/en.javascript.info#3726]
I remember comments were still visible at the end of each page of the tutorial a few days ago. Now, there is nothing.

![image](https://github.com/user-attachments/assets/6cf1f6b1-2903-42d9-a7ce-19188a4f06e9)

As a side effect, the "comments" link on the naviguation menu on the left does not do anything.

**Comment by jsghlink** [comment]
Consistent with the experience of @Gwizdo51, I stopped seeing the comments - first in the section 5.5 - Array methods yesterday and then in the majority of the lessons I happened to click on randomly. This was especially puzzling since I was able to view them perfectly well day before yesterday. There are still sections where they are visible (section 5.4 - Arrays, for example), and I am attaching some relevant screenshots to help you inform more about the issue.

![Screenshot_2024-07-30_01-11-27](https://github.com/user-attachments/assets/fd3a1d19-cd52-47f8-8e90-097fa41c51dd) - section 5.5 - Array methods
![Screenshot_2024-07-30_01-00-33](https://github.com/user-attachments/assets/0c146934-42b4-46ef-af04-4e0da316ecae) - section 5.4 - Arrays
![Screenshot_2024-07-30_01-31-34](https://github.com/user-attachments/assets/377fcb92-17b3-468c-9fa2-c9fd4ccc2fe4) - section 5.4 - Arrays (The topmost comment by "J S" is actually me trying to test if the comments section was working.)

I sincerely request @iliakan , @tadjik1 and the javascript.info for please looking into this issue and helping bring the comments back. 

They have been a great place to get exposed to others' ideas and have been an immense help in facilitating learning about concepts and implementations after completing the chapter(s).

I speak for everyone when I say a massive thank you to the whole javascript.info team for bringing this resource to the countless willing, learning souls and helping them learn in a structured way about JavaScript. 

A swift response to the issue will be appreciated by everyone and will be in the best interests of all.

[I first reached out on the Discord channel where it was suggested that I share my experience here and help bring this issue to your notice.]

**Comment by iliakan** [comment]
I removed disqus from the website.

Their ads are really nasty.
And most of the comments were... not helping.

Would you suggest an alternative commenting mechanism, perhaps?

**Comment by jsghlink** [comment]
Does that mean that I(or anyone else) will not be able to access the comments the old comments at all?

Personally, going through the comments section after completing a chapter has really helped accelerate my learning, I believe. On this particular occasion, in section (5.5 - Array methods), I was looking forward to try out some solutions to exercises suggested by certain users with posting some myself for some remarks. 

I get you point about the ads, though.

I'll have a think about suggesting any alternatives, but is self-hosting an option? In that case, you could try FOSS solutions like Isso or Cusdis (found these in a quick search).

Also, I think that it would have been best if the users would have been informed about this beforehand.

**Comment by iliakan** [comment]
I can self-host. 
I can't write a new commenting system from scratch.
Also I can't pay obscene amount of money to an external commenting system, because the website is free and has many pageviews / comments.

**Comment by jsghlink** [comment]
> Does that mean that I(or anyone else) will not be able to access the comments the old comments at all? 

Can you shed some light on this, please.

**Comment by iliakan** [comment]
This is correct.

**Comment by jsghlink** [comment]
From a quick look, there seems no way to get specifically to the comments on javasccript.info through disqus. I'll try asking someone on their forums maybe.

Thank you getting back, and take care!

**Comment by joaquinelio** [comment]
I liked disqus. 

With self-hosting,
SPAM will be a problem.

Disqus do the job of blocking 
spammers AND TROLLS due its account system

I know of 3rd party antispam, but im not sure you can defend yourself from troll BS without moderators.

**Comment by joaquinelio** [comment]
Mmm...

It navigates faster too.

But I like comments...

**Comment by iliakan** [comment]
May it be that there (free or fixed-reasonable-monthly fee) are alternatives to disqus?

**Comment by atakndogn** [comment]
Yes, there were some nonsensical comments, but also numerous inspiring and diverse code snippets to learn from for both newcomers and experienced programmers. Therefore, I respectfully request the return of the comments section.

**Comment by iliakan** [comment]
@atakndogn Suggest a suitable alternative to disqus, and I'll use it instead and transfer comments.

**Comment by joaquinelio** [comment]
As SP maintainer,

I liked the compliments to the maintainers part.

Sorry couldn't resist ✌️

**Comment by skkuraki** [comment]
it's half year, still no comments.

**Comment by javascriptru** [comment]
Indeed, no alternative was suggested.

**Comment by iliakan** [comment]
I just re-enabled the comments. 
We'll be in talks with disqus to see if the ads become more acceptable.

If disqus won't work for us, then we'll switch to another commenting system, I have a couple in mind! ;)
